### PR TITLE
AI Editor: add example config files

### PR DESCRIPTION
**Pull request by miltondp**

### CODE REVIEW

File: content/ai_revision-config.yaml
[the remark this answers is not in the document]
Comment (miltondp) [reply content/ai_revision-config.yaml:1]: Thanks, Daniel. I'll talk with @vincerubinetti and Faisal about this.

[the remark this answers is not in the document]
Comment (falquaddoomi) [reply content/ai_revision-config.yaml:1]: Hey @dhimmel, agreed; per your suggestion we've moved the config files to `ci` rather than `content`.

**Comment by miltondp** [comment]
@dhimmel this PR is ready to be merged. @vincerubinetti maybe you can start testing this now.

File: ci/ai-revision-prompts.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    Revised paragraph:
+
+  default: |
+    Proofread the following paragraph that is part of a scientific manuscript.
+    Keep all Markdown formatting, citations to other articles, mathematical
+    expressions, and equations.
+
+    Input paragraph: {{ content }}
+
+    Proofread paragraph:
```
Comment (dhimmel) [inline ci/ai-revision-prompts.yaml:44]: missing final newline.

Comment (miltondp) [reply ci/ai-revision-prompts.yaml]: Fixed!

**Review by dhimmel — APPROVED** [review]

**Comment by vincerubinetti** [comment]
@dhimmel note that doing a fresh fork of rootstock failed (for the ai editor) because [the manubot package is specified as a specific commit rather than a semver or github@latest version](https://github.com/CU-DBMI/rootstock/blob/main/build/environment.yml#L24). That will be need to be updated as well. It also seems like a new PIPY version should be pushed, and that rootstock should be based off of PYPI, but I'll leave that to you.

**Comment by miltondp** [comment]
> @dhimmel note that doing a fresh fork of rootstock failed (for the ai editor) because [the manubot package is specified as a specific commit rather than a semver or github@latest version](https://github.com/CU-DBMI/rootstock/blob/main/build/environment.yml#L24). That will be need to be updated as well. It also seems like a new PIPY version should be pushed, and that rootstock should be based off of PYPI, but I'll leave that to you.

From this, I understand that following the instructions to create a new Manubot-based manuscript (fork of this repo) and run the AI Editor would fail, right?

**Comment by dhimmel** [comment]
> note that doing a fresh fork of rootstock failed (for the ai editor) because the manubot package is specified as a specific commit rather than a semver or github@latest version.

This pull request should have incremented the manubot version for `manubot ai-revision --config-directory` support. My bad for not noting that during the review.

https://github.com/manubot/rootstock/pull/513 upgrades the manubot version used in rootstock and should resolve the problem.

I see the following as separate points as separate from the cause of this issue (and separate from eachother):

- whether rootstock installs the latest manubot version or a pinned version (I strongly prefer pinned here)
- whether the manubot package is installed from GitHub or PyPI

**Pull request merged**